sdk/nodejs/Makefile

```diff
@@ -17,10 +17,13 @@ include ../../build/common.mk
 
 export PATH:=$(shell yarn bin 2>/dev/null):$(PATH)
 
-lint::
+ensure::
+	yarn
```

Comment:
Is this a shortcut for yarn install?

Reply:
So actually this now makes yarn install run twice `make ensure`:

```
[nix-shell:~/pulumi/sdk/nodejs]$ make ensure
make ensure
ENSURE:
yarn install
yarn install v1.22.17
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.21s.
yarn
yarn install v1.22.17
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
✨  Done in 0.21s.
```

Reply:
Perhaps remove the relevant line from common.mk?

```
ensure::
	$(call STEP_MESSAGE)
	@if [ -e 'package.json' ]; then echo "yarn install"; yarn install; fi
``

Reply:
Will check if this line in common.mk can be removed or if we have other places it's causing a `yarn` to be run.